config/dev.webpack.config.js

```diff
@@ -19,12 +19,20 @@ plugins.push(
             './RootApp': resolve(__dirname, '../src/DevEntry')
         },
         exclude: [ 'react-redux' ],
-        shared: [{
-            'react-redux': {
-                requiredVersion: '*',
-                singleton: true
+        shared: [
+            {
+                'react-router-dom': {
+                    singleton: true,
+                    requiredVersion: '*'
+                }
+            },
+            {
```

Comment:
I don't think you want to do this. Sharing redux instances might introduce issues for you.